rero/rero-ils
# US716 holdings level

* Adds holdings level between documents and items.
* Adds automatic holding creation for new items based on item type and item location.
* Adds automatic attaching of new items to a holding record if a valid holding record exists.
* Adds automatic holding deletion once last item is deleted.
* Adds automatic re-linking if necessary to a new holding when item is updated.
* Adds holding structure on documents ES.
* Removes items structure on documents ES.
* Improves label pour one or more items link.
* Filters result with Holdings information.

Co-Authored-by: Bertrand Zuchuat <bertrand.zuchuat@rero.ch>
Co-Authored-by: Aly Badr <aly.badr@rero.ch>

# To test:
* bootstrap, setup
* create an item and make sure holding field is added to the item record
* display the created holding record at 'http://localhost:5000/global/holdings/xx '
* edit the item and change the item type or the location of the item
    ** make sure the item is re-attached to another holding.
* delete the item and make sure the holding is also deleted by checking '/http://localhost:5000/global/holdings/xx '
    ** (if the item is the last attached to this holding)
